packages/after/lib/loadInitialProps.js

```diff
@@ -1,19 +1,19 @@
 import { matchPath } from 'react-router-dom';
 
 export default function loadInitialProps(routes, pathname, ctx) {
-  return Promise.all(routes.filter(route => {
     const match = matchPath(pathname, route);
```

Comment:
where is route coming from ?

Reply:
next commit should be better. i need to run this on my machine next time, sorry I didn't think it through when i made the pr... my bad